R/support.R

```diff
@@ -393,3 +410,43 @@ combine_rt1 <- function(what, rt, samples) {
 
   Reduce(`+`, ret)
 }
+
+
+combine_rt1_EpiEstim <- function(what, rt, samples) {
```

Comment:
This function is never used so can be removed

Reply:
have added a test for a function which calls this one